src/NuGet.Core/NuGet.Protocol/EnhancedHttpRetryHelpers.cs

```diff
@@ -2,103 +2,110 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using NuGet.Common;
 
 namespace NuGet.Protocol
 {
     internal class EnhancedHttpRetryHelper
     {
-        private readonly IEnvironmentVariableReader _environmentVariableReader;
-        internal const string ExperimentalRetryEnabledEnvVarName = "NUGET_ENABLE_EXPERIMENTAL_HTTP_RETRY";
-        internal const string ExperimentalRetryTryCountEnvVarName = "NUGET_EXPERIMENTAL_MAX_NETWORK_TRY_COUNT";
-        internal const string ExperimentalRetryDelayMsEnvVarName = "NUGET_EXPERIMENTAL_NETWORK_RETRY_DELAY_MILLISECONDS";
+        /// <summary>
+        /// The default delay in milliseconds between retries.
+        /// </summary>
+        public const int DefaultDelayMilliseconds = 1000;
 
-        public EnhancedHttpRetryHelper(IEnvironmentVariableReader environmentVariableReader)
-        {
-            _environmentVariableReader = environmentVariableReader ?? throw new ArgumentNullException(nameof(environmentVariableReader));
-        }
+        /// <summary>
+        /// The default value indicating whether or not enhanced HTTP retry is enabled.
+        /// </summary>
+        public const bool DefaultEnabled = true;
+
+        /// <summary>
+        /// The default number of times to retry.
+        /// </summary>
+        public const int DefaultRetryCount = 6;
+
+        /// <summary>
+        /// The environment variable used to change the delay value.
+        /// </summary>
+        public const string ExperimentalRetryDelayMsEnvVarName = "NUGET_EXPERIMENTAL_NETWORK_RETRY_DELAY_MILLISECONDS";
+
+        /// <summary>
+        /// The environment variable used to enable or disable the enhanced HTTP retry..
+        /// </summary>
+        public const string ExperimentalRetryEnabledEnvVarName = "NUGET_ENABLE_EXPERIMENTAL_HTTP_RETRY";
```

Comment:
should the env var and property still be called "experimental" given it's validated and on by default now? might be better to call it `NUGET_ENABLE_ENHANCED_HTTP_RETRY`

Reply:
+1 to this, additionally should there be a doc somewhere that contains the variable name info and usage now that it's no longer "experimental"? (I only saw it mentioned in 6.0 release notes)

Reply:
Sounds like a good idea to rename the env vars, @MattGal are you okay with not supporting the old env vars anymore?  When this ships, you'll start getting the defaults instead of your custom values until you update build logic.  Or I can make this backwards compatible with the previous env var names.

Reply:
> , @MattGal are you okay with not supporting the old env vars anymore? 
Given the defaults are the only values the teams I know using the values are setting, I am OK with this